modules/sharedaddy/sharing-sources.php

```diff
@@ -266,11 +266,20 @@ public function get_amp_display( $post ) {
 	 * @param array $attrs Custom attributes for rendering the social icon.
 	 */
 	protected function build_amp_markup( $attrs = array() ) {
+
+		$title = sprintf(
+			/* translators: placeholder is a service name, such as "Twitter" or "Facebook". */
+			__( 'Click to share on %s', 'jetpack' ),
+			$this->get_name()
+		);
+
 		$attrs        = array_merge(
 			array(
-				'type'   => $this->get_id(),
-				'height' => '32px',
-				'width'  => '32px',
+				'type'			=> $this->get_id(),
+				'height'		=> '32px',
+				'width'			=> '32px',
+				'aria-label'	=> $title,
+				'title'			=> $title,
```

Comment:
You'll want to use spaces inside inline structures like this one, as per [the coding standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#indentation):

```suggestion
				'type'       => $this->get_id(),
				'height'     => '32px',
				'width'      => '32px',
				'aria-label' => $title,
				'title'      => $title,
```

I'm actually a bit surprised you were able to commit this; I would expect our pre-commit hook to block you with an error like this one:

```
> vendor/sirbrillig/phpcs-changed/bin/phpcs-changed --git 'modules/sharedaddy/sharing-sources.php'

FILE: modules/sharedaddy/sharing-sources.php
-----------------------------------------------------------------------------------------------
FOUND 1 ERROR AND 0 WARNINGS AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------
 282 | ERROR | Spaces must be used for mid-line alignment; tabs are not allowed
-----------------------------------------------------------------------------------------------
Script vendor/sirbrillig/phpcs-changed/bin/phpcs-changed --git handling the php:changed event returned with error code 1
COMMIT ABORTED: The linter reported some problems. If you are aware of them and it is OK, repeat the commit command with --no-verify to avoid this check. But please don't. Code is poetry.
husky > pre-commit hook failed (add --no-verify to bypass)
```

Reply:
Thanks @jeherve!